imports/api/employee.js

```diff
@@ -0,0 +1,234 @@
+import { Meteor } from "meteor/meteor";
+import { Employees, Tasks } from "./db";
+import EMPLOYEESAPI from "../constant/methods/employeesAPI";
+import { isAuthenticated } from "../util/authUtil";
+import { AuthError, EmployeeError } from "../constant/error";
+import { EmployeeMessage } from "../constant/message";
+import { removeElement, addToList } from "../util/arrayUtil";
+import { isAdmin, removeRelatedTasks } from "../util/employeeUtil";
+
+if (Meteor.isServer) {
+	Meteor.publish("employees", () => Employees.find());
+}
+
+export const insertEmployee = (
+	db,
+	_id,
```

Comment:
lets not have a _id here since meteor mongodb generate _id when insert. And lets change employeeId to accountId since the _id in employees db is technically employeeId and the id from meteor account-base is accountId.